kcbq-connector/src/main/java/com/wepay/kafka/connect/bigquery/config/BigQuerySinkConfig.java

```diff
@@ -406,6 +406,13 @@ protected static void ensureValidMap(String name, List<String> values) {
     }
   }
 
+  /**
+   * Returns the keyfile
+   */
+  public String getKeyFile() {
+    return getPassword(KEYFILE_CONFIG).value();
```

Comment:
Will this NPE if no keyfile is specified?

Reply:
No, it calls 
```public Password getPassword(String key) {
        return (Password) get(key);
    }
``` 
which calls 
```
protected Object get(String key) {
        if (!values.containsKey(key))
            throw new ConfigException(String.format("Unknown configuration '%s'", key));
        used.add(key);
        return values.get(key);
    }
``` 
so we should be safe if the config is empty

Reply:
We throw an exception upstream if the config is null/empty, so I think that should prevent the NPE?